loolwsd-systemplate-setup

```diff
@@ -21,6 +21,16 @@ CHROOT=`cd $CHROOT && /bin/pwd`
 INSTDIR_LOGICAL=`cd $INSTDIR && /bin/pwd -L`
 INSTDIR=`cd $INSTDIR && /bin/pwd -P`
 
+if [ ! `uname -s` = "FreeBSD" ]; then
+    CP=cp
+    REALPATH=realpath
+    LOCALBASE=usr/
+else
+    CP=gcp
```

Comment:
Freebsd doesn't have `cp`? (I find it hard to believe.)

Reply:
It does, but it has different flags. Some like `--dereference` are missing. So I thought it is simpler to GNU `cp`.